forecasting/src/autogluon/forecasting/learner/abstract_learner.py

```diff
@@ -0,0 +1,157 @@
+import os
+import random
+
+from gluonts.evaluation import Evaluator
+
+from autogluon.core.utils.loaders import load_pkl
+from autogluon.core.utils.savers import save_pkl, save_json
+
+from ..trainer import AbstractTrainer
+
+
+class AbstractLearner:
+
+    learner_file_name = 'learner.pkl'
+    learner_info_name = 'info.pkl'
+    learner_info_json_name = 'info.json'
+
+    def __init__(self, path_context: str, eval_metric=None, is_trainer_present=False, random_seed=0):
+        self.path, self.model_context, self.save_path = self.create_contexts(path_context)
+
+        if eval_metric is not None and eval_metric not in ["MASE", "MAPE", "sMAPE", "mean_wQuantileLoss"]:
```

Comment:
@dcmaddix for now I believe this is since although evaluation is outsourced totally to GluonTS, whether "less/more is better" in the loss is encoded in `utils.metric_utils`. I made this more modular so all evaluation checks refer to this module for available metrics, and new metrics can be added easily there.